frontend/src/bundles/auth/components/auth-layout/auth-layout.tsx

```diff
@@ -12,23 +12,37 @@ const AuthLayout: React.FC<Properties> = ({ children }) => {
         <>
             <Grid container className={styles.container}>
                 <Grid item xs={12} md={6}>
-                    <Grid item className={styles['selling-point']}>
+                    <Grid item className={styles.sellingPoint}>
                         <div className={styles.logo}></div>{' '}
                         {/* TODO: Change to Logo component*/}
-                        <div className={styles.message}>
-                            <div
-                                className={getValidClassNames(
-                                    styles['message-1'],
-                                    styles.text,
-                                )}
-                            >
-                                Find the top talent for your business{' '}
-                                {/* TODO: Animate message */}
-                            </div>
-                        </div>
+                        <Typography
+                            className={styles.animatedTextWrapper}
+                            variant="h1"
+                        >
+                            <Grid className={styles.message}>
+                                <span
+                                    className={getValidClassNames(
+                                        styles.messageOne,
+                                        styles.text,
+                                    )}
+                                >
+                                    <span className={styles.text}>
+                                        Find the top talent{' '}
```

Comment:
```suggestion
                                        Find the top talent
```
Why you have {' '} in component?

Reply:
Linter inserts it.